esp/scm/ws_workunits_struct.ecm

```diff
@@ -528,8 +528,8 @@ ESPStruct WUFileOption
     int SlaveNumber(1); //ThorSlaveLog: optional
     int64 SizeLimit(0);
     [min_ver("1.90"), nil_remove] unsigned MaxLogRecords;
-    [min_ver("1.90"), nil_remove] ESPenum LogSelectColumnMode LogSelectColumnMode;
-    [min_ver("1.90"), nil_remove] ESPenum LogAccessLogFormat LogFormat;
+    [min_ver("1.90"), nil_remove] ESPenum LogSelectColumnMode LogSelectColumnMode(0);
```

Comment:
Are these correct?  Shouldn't this be using the associated enumeration text e.g., "DEFAULT" (and say "XML" for LogFomat?)

Reply:
Would 1 be a more appropriate default?